### PR TITLE
Docs(web-react, web): Add Spinner demo #DS-907

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Spinner demos are now same in all packages

### Issue reference

[Demo: Spinner má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-907)

### CODE REVIEW

**Comment by literat** [comment]
I think here is the same problem with missing `docs-Stack`.

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-react/src/components/Spinner/demo/SpinnerColors.tsx [hunk 1]
```diff
@@ -1,34 +1,23 @@
-// Because there is no `dist` directory during the CI run
-/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
 import React from 'react';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore: No declaration file
-import icons from '@lmc-eu/spirit-icons/dist/icons';
-import { IconsProvider } from '../../../context';
 import Spinner from '../Spinner';
+import DocsBox from '../../../../docs/DocsBox';
```
Comment (crishpeen) [inline packages/web-react/src/components/Spinner/demo/SpinnerColors.tsx:3]: 🔠

File: packages/web-react/src/components/Spinner/demo/index.tsx [hunk 2]
```diff
@@ -7,18 +7,22 @@ import ReactDOM from 'react-dom/client';
 import icons from '@lmc-eu/spirit-icons/dist/icons';
 import { IconsProvider } from '../../../context';
 import DocsSection from '../../../../docs/DocsSections';
```
Comment (crishpeen) [inline packages/web-react/src/components/Spinner/demo/index.tsx:9]: abc?

File: packages/web/src/scss/components/Spinner/index.html [hunk 3]
```diff
@@ -1,35 +1,70 @@
...
 
 <section class="docs-Section">
-  <h2 class="docs-Heading">Spinner</h2>
 
-  <div>
+  <h2 class="docs-Heading">Default</h2>
+  
+  <div class="docs-Stack docs-Stack--start">
+
     <svg width="24" height="24" class="animation-spin-clockwise">
```
Comment (crishpeen) [inline packages/web/src/scss/components/Spinner/index.html:9]: we are probably missing aria-hidden=true everywhere, can you please add it? It is a default in the Icon component.

**Review by crishpeen — APPROVED** [review]

File: packages/web-react/src/components/Spinner/demo/index.tsx (hunk 2 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Spinner/demo/index.tsx:8]: sorted

File: packages/web-react/src/components/Spinner/demo/SpinnerColors.tsx (hunk 1 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Spinner/demo/SpinnerColors.tsx:2]: sorted

File: packages/web/src/scss/components/Spinner/index.html (hunk 3 above)
Comment (pavelklibani) [reply packages/web/src/scss/components/Spinner/index.html:9]: we agreed there will be separate pull request adding this everywhere

**Pull request merged**